R/translate-sql-odbc.R

```diff
@@ -9,15 +9,30 @@ base_odbc_scalar <- sql_translator(
   as.logical    = sql_cast("BOOLEAN"),
   as.character  = sql_cast("STRING"),
   as.Date       = sql_cast("DATE"),
-  round         = function(x, digits = 0L)
-                    build_sql(
-                      "ROUND(", x, ", ", as.integer(digits),")"
-                      ),
   paste0        = sql_prefix("CONCAT"),
-  paste         = function(..., sep = " ")
-                    build_sql(
-                      "CONCAT_WS(",sep, ", ",escape(c(...), parens = "", collapse = ","),")"
-                      )
+  # cosh, sinh, coth and tanh calculations are based on this article
+  # https://en.wikipedia.org/wiki/Hyperbolic_function
+  cosh = function(x){
+    build_sql("(EXP(", x, ") + EXP(-", x,")) / 2")
+  },
+  sinh = function(x){
+    build_sql("(EXP(", x, ") - EXP(-", x,")) / 2")
```

Comment:
I think you need more parens: i.e. if x is "a + b", you don't want `EXP(-a + b)`